packages/@webex/plugin-meetings/src/meetings/util.ts

```diff
@@ -49,6 +51,16 @@ MeetingsUtil.handleRoapMercury = (envelope, meetingCollection) => {
     if (meeting) {
       const {seq, messageType, tieBreaker, errorType, errorCause} = data.message;
 
+      Metrics.sendBehavioralMetric(BEHAVIORAL_METRICS.ROAP_MERCURY_EVENT_RECEIVED, {
+        correlation_id: data.correlationId,
+        seq,
+        message_type: messageType,
+        sdp: data.message.sdps?.length > 0 ? data.message.sdps[0] : undefined,
+        tie_breaker: tieBreaker,
```

Comment:
tie breaker for sure won't be needed for anything in metrics

Reply:
removed